src/main/java/org/bytedeco/javacpp/tools/Info.java

```diff
@@ -124,6 +129,14 @@ public Info(Info i) {
     String cppText = null;
     /** Outputs the given code, instead of the result parsed from the declaration of C++ identifiers. */
     String javaText = null;
+    /** List of cpp names (class, struct, member function) the declarations of which we want to copy in this class or struct. */
+    List<String> copyFrom = null;
+    /** If not-null, declarations for these cpp names are copied.
+     * Set to non-null by InfoMap.normalizeCopy or for polymorphic classes. */
+    CopiedDeclarations copiedDeclarations = null;
+    /** Set by Parser when a class has a virtual member function or inherit from a polymorphic class. */
+    boolean polymorphic = false;
```

Comment:
Info is meant to contain configuration values from the user. If those are not meant to be set by the user, please find another place for them.

Reply:
Done, it's better like this (although `Info` is already used to store things not set by user (like constructors text)).

Reply:
Right, it does store some stuff by default, but that can be overridden by the user, or at least that's the intention and if it doesn't allow it it's probably a bug.